x-pack/plugins/security_solution/public/resolver/store/data/reducer.ts

```diff
@@ -7,34 +7,53 @@
 import { Reducer } from 'redux';
 import { DataState } from '../../types';
 import { ResolverAction } from '../actions';
+import * as treeFetcherParameters from '../../models/tree_fetcher_parameters';
 
 const initialState: DataState = {
   relatedEvents: new Map(),
   relatedEventsReady: new Map(),
   resolverComponentInstanceID: undefined,
+  tree: {},
```

Comment:
The data related to fetching the resolver tree is now namespaced under `tree`.

Reply:
I'll likely try to move this logic into a separate nested selector in the future.